app/content/models/registration.py

```diff
@@ -51,30 +52,34 @@ class Meta:
 
     @classmethod
     def has_retrieve_permission(cls, request):
-        return check_has_access(cls.has_retrieve_access, request,)
+        return True
 
     @classmethod
     def has_list_permission(cls, request):
-        return check_has_access(cls.has_access, request,)
+        return cls.has_event_permission(cls, request)
 
-    @staticmethod
-    def has_write_permission(request):
+    @classmethod
+    def has_write_permission(cls, request):
         return bool(request.user)
 
-    @staticmethod
-    def has_create_permission(request):
-        return request.id is not None
+    def has_event_permission(self, request):
+        if request.user is None:
+            return False
+
+        event = Event.objects.get(id=request.parser_context["kwargs"]["event_id"])
```

Comment:
Denne føles ikke som den hører hjemme her. Sjekker du på arrangementet tilknyttet påmeldingen? I så fall
```suggestion
        event = self.event
```

Burde også flyttes til `Event` for bedre cohesion